ci/terraform/oidc/spot-sqs.tf

```diff
@@ -56,7 +98,7 @@ resource "aws_sqs_queue_policy" "spot_request_queue_policy" {
   ]
```

Comment:
was tempted to get rid of this depends on as it seems a bit wrong... but obviously don't want to break anything spot related!

Reply:
This does seem wrong - if we do keep should we add `cross_account_spot_request_queue_policy_document` . If there was a need for this feels like `cross_account_spot_request_queue_policy_document`  would be important.

Reply:
on chatting with ethan, going to remove the depends on and pause promotion beyond staging to test